src/components/Card.jsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```

Comment:
Still looks like the card component is on this PR. Let's figure out a way to remove this file from the PR. Let me know if you need help with this.